website/content/en/docs/concepts/nodeclasses.md

```diff
@@ -56,6 +56,9 @@ spec:
   amiSelectorTerms:             
```

Comment:
One thing to callout in our docs changes. This shouldn't go in `docs` (I know it's confusing) because this is the most recently _released_ version of our docs. This belongs in preview since we promote that version of the docs to the latest release version when we are ready to ship it.

Reply:
Gotcha, I follow :+1: moved into `/preview` in https://github.com/aws/karpenter-provider-aws/commit/bb62cc9c65b4fd39dd5b3fcd26059e352a592028. There's still that new [example](https://github.com/aws/karpenter-provider-aws/blob/bb62cc9c65b4fd39dd5b3fcd26059e352a592028/examples/v1beta1/instance-store-ephemeral-storage.yaml) though -- should it be moved/removed/commented?

Reply:
The example is fine where it's at 👍🏼